antenna/ext/pubsub/crashpublish.py

```diff
@@ -2,11 +2,6 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
-# The pubsub code needs monkeypatching to happen before it's imported.
-# This guarantees that.
-from gevent import monkey
-monkey.patch_all()  # noqa
```

Comment:
This was an errant fix. The theory was that somehow the monkey patching wasn't happening before we imported pubsub things. That turned out not to be the issue but rather that the monkey patching didn't affect grpc which is not a pure Python library. Thus this fix had no effect.